plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/portforwarding/PortForwardingUtils.java

```diff
@@ -53,15 +53,11 @@ public static boolean isPortInUse(int port) {
 			return false;
 		}
 		try (ServerSocket socket = new ServerSocket(port)){
-			//not in use apparently
+			//not in use
+		    return false;
 		} catch (IOException e) {
 			return true;//in use
 		}
-		try (Socket socket = new Socket("localhost", port)) {
-			return true; // in use
-		} catch (IOException e) {
-			return false;
-		}
```

Comment:
this code looks obscure, trying to allocate a port twice, once with host, once without. Any idea why this was done? Is this maybe something that's required on specific platforms (Mac?). 
I'd suggest that we test this on all 3 platforms before merging.